py_sorter/sorter.py

```diff
@@ -30,7 +30,7 @@ def parse_args(args):
     sorting_group = parser.add_mutually_exclusive_group(required=True)
     sorting_group.add_argument('-s', '--sort', type=str,
                                choices=['bubble', 'bogo', 'merge', 'selection', 'quick', 'radix', 'insertion',
-                                        'insertion_recursive', 'heap'],
+                                        'insertion_recursive', 'heap','shell'],
```

Comment:
Use consistent format in regards to any commas in lists.

```suggestion
                                        'insertion_recursive', 'heap', 'shell'],
```